app/scripts/views/forms.coffee

```diff
@@ -971,6 +972,26 @@ define [
       @listenToNewFormView()
       @showNewFormViewAnimate()
 
+
+    # Request the history of the form model, the user can click on a revision and see the details
+    # preferably by simply showing the other revision as a model along side in the same views
+    fetchHistory: (formModel) ->
+
+      fielddbHelperModel = new (FieldDB.Datum)(_id: formModel.id)
+      fielddbHelperModel.fetchRevisions().then(((revisions) ->
```

Comment:
It's fielddb#2.45.3. So I have to merge Feature fielddb2.50 integration #118 before I can make this work.